src/components/pages/Restaurant.test.tsx

```diff
@@ -0,0 +1,152 @@
+import { render, screen } from '@testing-library/react'
+import TestProvider from '../partials/TestProvider'
+import { UserContext } from '../partials/UserContext'
+import Restaurant from './Restaurant'
+
+let mockReviews = [{}]
```

Comment:
set type for mockReviews and we can easily check any issue when the review model is changed.

Reply:
fixed in the current PR I am working on. Will mention in comment when I submit that one. thanks!